api/api.go

```diff
@@ -695,6 +701,25 @@ func procDetachBackend(c libnetwork.NetworkController, vars map[string]string, b
 	return nil, &successResponse
 }
 
+func procUpdateSandbox(c libnetwork.NetworkController, vars map[string]string, body []byte) (interface{}, *responseStatus) {
```

Comment:
Since this is a public PUT API, can add a check to make it explicit we disallow changing container ID. We can return forbidden, instead of ignoring it now.